ci-operator/step-registry/jira-agent/ONBOARDING.md

```
@@ -41,83 +41,59 @@ For teams working on repos within the `openshift/` GitHub org, you'll typically
-
-      # Optional: credential key names in your Vault secret
-      JIRA_AGENT_UPSTREAM_INSTALLATION_ID_KEY: "upstream-installation-id"
-      JIRA_AGENT_FORK_INSTALLATION_ID_KEY: "fork-installation-id"
-
-      # Optional: project-specific tool/plugin setup (runs before processing)
-      JIRA_AGENT_TOOL_SETUP_SCRIPT: "GOFLAGS='' go install golang.org/x/tools/gopls@latest"
-
-      # Optional: code review configuration
-      JIRA_AGENT_REVIEW_LANGUAGE: "go"
-      JIRA_AGENT_REVIEW_PROFILE: ""
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)